### ISSUE TITLE
BUG: realtime query conditions ignored, all datas returned

### ISSUE THREAD

**Issue by 8byr0** [spaceuptech/space-cloud#190]
When using realtime from js SDK, the behavior is not correct.

# How to reproduce
```
db.liveQuery("messages")
        .where(
            cond("to", "==", 'aStringThatDoesNotExist')
        )
        .subscribe(onMessages, onError)
```
## Actual behavior
The success callback is triggered each time a new document is added to messages collection. The parameters passed to this cbk contain all documents, no matter the condition.

## Expected behavior
The callback should receive only documents matching the condition and potentially be triggered only if the condition matches.

For those who would be interested in a quick and dirty fix:
## If you face the same issue
Before opening this issue, I was trying to do this livequery: 
```javascript
const startMessagesRealtime = (partnerID, onMessages, onError) => {
    const user = JSON.parse(localStorage.getItem("user"));

    // Get todos from the database
    db.liveQuery("messages")
        .where(
            or(
                and(cond("to", "==", user._id), cond("from", "==", partnerID)),
                and(cond("to", "==", partnerID), cond("from", "==", user._id))
            )
        )
        .subscribe(onMessages, onError)

}
```
As condition is ignored, you can filter elements in the `onMessage` callback

### Using lodash
```javascript
import * as _ from 'lodash';
// Inside your callback
const messages = _.reject(docs, message => (message.to !== elt.user._id) && (message.from !== elt.user._id) );
```

### Plain JavaScript
```javascript
// Inside your callback
const messages = docs.filter( message => (message.to !== elt.user._id) && (message.from !== elt.user._id) );

**Comment by YourTechBud** [comment]
I couldn't reproduce this. Seems to work properly on the real-time todo example.

How many times are you calling liveQuery? Are you unsubscribing when it isn't needed?

**Comment by YourTechBud** [comment]
Resolved in spaceuptech/space-api-js#28
